users/crashrep.rst

```diff
@@ -39,4 +39,12 @@ The following is an example of a crash report as sent::
     main.syncthingMain(0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x49651b4, 0x1, 0x0, 0x0, ...)
             /buildagent/work/github.com/syncthing/syncthing/cmd/syncthing/main.go:873 +0x1bc5
 
+Note that when using a custom compiled build, the names of the domain,
+computer, and user of the machine where Syncthing was built will be
+included in the log as part of the version string. This information is
+essential for the developers to interpret the crash log in context. If
+you compile Synching locally and want to prevent your build from having
+such data embedded, set the ``COMPUTERNAME``, ``USERDOMAIN``, and
```

Comment:
I'm not sure where these variables come from but they don't appear to be what we actually look at


https://github.com/syncthing/syncthing/blob/a85bc1c3a64e33f1c42fe6e1dbeb19a953d01431/build.go#L999

Reply:
There should be instructions on tweaking the compiled in metadata on the build page, and we should link to that from here rather than duplicating, I think

Reply:
We can refer to https://docs.syncthing.net/dev/building.html#version-tagging by

```
:ref:`versiontagging`
```

Reply:
The variables come from https://forum.syncthing.net/t/15417/7. I must admit though that I have not tested them myself, but rather just believed the word :hushed:.

I will try to make all the changes shortly, but let me first actually compile and test everything myself just to make sure that there are no further similar flops on my part.

Reply:
I have simplified and rewritten the note to match the terminology used in https://docs.syncthing.net/dev/building.html#version-tagging. I have also tested the compilation, and indeed the variables listed by @calmh are the right ones, while the other ones do not do anything.